lib/tasks/data/export.rake

```diff
@@ -6,32 +6,51 @@ namespace :data do
 
       admin = User.find 1
       scores_model_by_id = {
-        56 => Course56ProjectScore,
-        57 => Course57ProjectScore,
-        58 => Course58ProjectScore,
         71 => Course71ProjectScore,
         73 => Course73ProjectScore
```

Comment:
do we need bravenx here too?

Reply:
We never ran the export for BravenX. Once we get to the point of exporting grades for BravneX (and Lehman), we'll need to create tables to dump into and work with the data team to write dashboards for them.